### PR TITLE
Fix special char usage in cmd line args for Rate Limiting the PRs

### PR DESCRIPTION
**Pull request by tarishij17**

### CODE REVIEW

File: dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimiter.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public RateLimiter() {
      * @see net.sourceforge.argparse4j.inf.Namespace Namespace
      */
     public static RateLimiter getInstance(Namespace ns) {
-        String rateLimitPRCreation = ns.get(Constants.RATE_LIMIT_PR_CREATION);
+        String rateLimitPRCreation = ns.get((Constants.RATE_LIMIT_PR_CREATION).replace("-","_"));
```
Comment (jeetchoudhary) [inline dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimiter.java:78]: I think the better thing is to use the underscore in the constant string and add a unit test to validate that no char is used in ArgumentParser that will be replaced automatically. 
Also, we need to test the changes once again.

Comment (afalko) [reply dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimiter.java:78]: What is the error that we are hitting if we don't make this change? Can we change the documentation is error out if invalid chars are passed?

Comment (jeetchoudhary) [reply dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimiter.java:78]: '-' is replaced by '_' by argparse4j ArgumentParser while parsing arguments. so "random-arg val" will be set to random_arg=val in the namespace.
What we are purposing here is 
1. change the argument using - to use _
2. add a unit test to throw the error if someone uses it unintentionally.

Comment (afalko) [reply dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/RateLimiter.java:78]: I see, so users won't see any difference, it is just how we parse internally?

**Review by afalko — APPROVED** [review]

**Review by jeetchoudhary — APPROVED** [review]
LGTM

**Pull request merged**